config/default.yml

```
@@ -1,4 +1,8 @@
 AllCops:
+  Excludes:
+    - 'bin/**'
+    - 'config/**'
```

```suggestion
    - 'config/**'
    - 'db/**'
```

How about `db` as well?

---

I don't think `db` should be excluded except for maybe `db/schema.rb` which is auto generated. The code in `db/migrations` is written by developers so should be linted.

---

I guess my enabling linting for old apps and going through and fixing old offenses (like `Style/StringLiterals`) in migrations has skewed my value judgement here!